src/main/java/core/basesyntax/service/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.strategy.impl.BalanceOperationService;
+import core.basesyntax.service.strategy.impl.PurchaseOperationService;
+import core.basesyntax.service.strategy.impl.ReturnOperationService;
+import core.basesyntax.service.strategy.impl.SupplyOperationService;
+import java.util.Map;
+
+public class OperationStrategy {
+    private final Map<FruitTransaction.Operation, OperationService> operationServiceMap = Map.of(
+            FruitTransaction.Operation.BALANCE, new BalanceOperationService(),
+            FruitTransaction.Operation.SUPPLY, new SupplyOperationService(),
+            FruitTransaction.Operation.PURCHASE, new PurchaseOperationService(),
+            FruitTransaction.Operation.RETURN, new ReturnOperationService());
```

Comment:
I'd recommend moving this map initialization to your main method and pass it to the constructor of this class